source/physics/elasticity/standard_tensors.inst.in

```diff
@@ -0,0 +1,41 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+for (deal_II_dimension : DIMENSIONS)
+{
+    template
+    class StandardTensors<deal_II_dimension>;
```

Comment:
It *instantiates* them, but only if there is a *definition*. That's what's missing for static member variables -- these are global variables in a sense, and you've declared them, but you still need to provide a *definition* for them that can then be instantiated.

For example, if you had
```
  template <int dim> class C {
    static const int var = 13;
  };
  template class C<1>;  // explicit instantiation, but the compiler has not seen a definition for C::var
```
what you need to do is this:
```
  template <int dim> class C {
    static const int var = 13;
  };
  template <int dim> const int C<dim>::var;  // definition
  template class C<1>;  // explicit instantiation, but the compiler has not seen a definition for C::var
```